x-pack/plugins/apm/ftr_e2e/cypress/e2e/service_map/service_map.cy.ts

```diff
@@ -50,25 +50,35 @@ describe('service map', () => {
   describe('when navigating to service map', () => {
     beforeEach(() => {
       cy.intercept('GET', '/internal/apm/service-map?*').as('serviceMap');
-      cy.visitKibana(serviceMapHref);
-
-      cy.wait('@serviceMap');
     });
 
-    it('shows nodes in service map', { retries: 3 }, () => {
+    it('shows nodes in service map', () => {
+      cy.visitKibana(serviceMapHref);
+      cy.wait('@serviceMap');
       cy.wait(500);
-      cy.getByTestSubj('serviceMap').matchImage();
+      cy.getByTestSubj('serviceMap').matchImage({
+        imagesPath: '{spec_path}/snapshots',
+        matchAgainstPath: 'cypress/e2e/service_map/snapshots/service_map.png',
+        maxDiffThreshold: 0.05, // maximum threshold above which the test should fail
```

Comment:
There's also an option in cypress to blackout specific elements. That would be better than increasing the threshold:

```suggestion
        screenshotConfig: {
  				blackout: ['.element-to-be-blackouted']
		  },
```